content/blog/2019-__-__-hydrology_task_view.md

```diff
@@ -0,0 +1,153 @@
+  filter(Imports %in% hyd_packages) %>% 
+  as_tbl_graph() 
+
+ggraph(hyd_tbl, layout = 'nicely') + 
+  geom_edge_link(colour = "grey50") + 
+  geom_node_text(aes(label = name), colour = "#ED7953FF", size = 6)
+```
+
+![](blog_post_files/figure-html/unnamed-chunk-8-1.png)<!-- -->
+
+Clearly some connectivity among water-related packages is already happening, which serves as a reminder that CRAN task views aren't the only place people discover packages to use. Some tools form a hub of connected packages because they share a common package maintainer (see `hydroGOF`) while others like `CityWaterBalance` draw upon a diverse suite of `Hydrology` packages to study urban water systems. Many rOpenSci packages have been included in the task view (`bomrang`, `dbhydroR`, `GSODR`, `hddtools`, `hydroscoper`,`MODIStsp`, `rnoaa`, `smapr`, `tidyhydat`, `weathercan`). `rnoaa` in particular is a key piece of infrastructure that underpins many analyses in hydrological research. The plot below summarises the number of downloads from the RStudio CRAN mirror over the past year. It is illustrative that the top four packages feature two modelling packages and two data packages. These are the basic tenets of hydrological research and indicate how those in the hydrological space are using R.   
+
+
+```r
+pkgcount <- cran_downloads(packages = hyd_packages, 
+                           from = Sys.Date()-1*365, 
+                           to = Sys.Date()) %>%
+  group_by(package) %>%
+  summarise(downloads = sum(count)) %>%
+  filter(downloads >= 5000)
+
+pkgcount %>%
+  ggplot(aes(x = reorder(package, downloads), y = downloads)) +
+  geom_col(aes(fill = downloads)) +
+  geom_text(aes(y = 1000, label = package), size = 5.5, colour = "white", 
+            hjust = 0, nudge_y = -500, angle = 90) +
+  scale_fill_viridis_c(name = "Downloads", option = "D", end = 0.75) 
+```
+
+![](blog_post_files/figure-html/countDownloads-1.png)<!-- -->
+
+
+## Future of R for Hydrology
+
+Hydrology, like many other sciences, is becoming more and more data driven and programmatic ways to access, manipulate and model data in a reproducible workflow are becoming a necessity.Water related data though has its own specific challenges, which are similar but different from those of other data sources. It can be expected that with a growing uptake of R in the hydrological community more packages will be available, with more interconnectivity between them. At the moment the landscape of R for hydrologists is quite fragmented, but in time it might be that some *core* packages to manipulate data will emerge and several computational approaches to hydrological models will be centralised in a unique package.Regardless of how the landscape of hydrology packages will evolve, R and open source tools are becoming more and more important in the hydrological community, and we are happy to have made a small contribution to make it easier for everybody to know what can already be done with water data in R. 
```

Comment:
add end of sentence space in 2 places: 

"necessity.Water " -> "necessity. Water "

"unique package.Regardless of how" -> "unique package. Regardless of how"